tests/integration/tier0/config-file/test_config_file.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+
+def test_config_file(convert2rhel):
+    paths = (
+        ("~/.convert2rhel_custom.ini", "[subscription_manager]\nactivation_key = config_activationkey"),
+        ("~/.convert2rhel.ini", "[subscription_manager]\npassword = config_password"),
+        ("~/password_file", "file_password"),
+    )
+
+    for path, content in paths:
+        f = open(os.path.expanduser(path), "w")
+        f.write(content)
+        f.close()
+        os.chmod(os.path.expanduser(path), 0o600)
+
+    with convert2rhel('--no-rpm-va --debug -c "~/.convert2rhel_custom.ini"') as c2r:
+        c2r.expect("DEBUG - Found activation_key in /root/.convert2rhel_custom.ini")
+        c2r.sendcontrol("c")
```

Comment:
I did small performance change - testing script isn't waiting for the first question and is stopped immediately after the debug messages. So the whole testing is faster.